www/js/constants.js

```diff
@@ -0,0 +1,38 @@
+angular.module('epaper')
+.constant('ShApiConstants', {
+	
+	/* 
+	 * Handling CORS issues for 'ionic serve'
+	 * 
+	 * Ref: http://blog.ionic.io/handling-cors-issues-in-ionic/
+	 * Proxied path is defined in ionic.config.json
+	 * 
+	 * Options:
+	 * - true: when need to test run using 'ionic serve' or 'ionic run -l'
+	 * - false: when test run using 'ionic run' or 'ionic emulate' or during UAT or Production
+	 */ 
+	useProxy : false,
```

Comment:
Usually we use "CORS" plugin at the browser to test the issue. But it is good also to use this out of the box function from ionic. Is it documented into the README?